### PR TITLE
Thieme.js -- fix eslint warnings

### PR DESCRIPTION
**Pull request by retorquere**

### CODE REVIEW

**Comment by retorquere** [comment]
updated for @zotero/eslint-config@1.0.2

File: Thieme.js [hunk 1]
```diff
@@ -37,32 +37,37 @@
...
-  Other example: 
+  Other example:
     https://www.thieme-connect.de/DOI/DOI?10.1055/s-2008-1081006
   which does not save as a test
 */
 
 
 // attr()/text() v2
-function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+function text(docOrElem, selector, index) {
```
Comment (zuphilip) [inline Thieme.js]: delete completely

File: Thieme.js [hunk 2]
```diff
@@ -77,22 +82,22 @@ function getSearchResults(doc, checkOnly) {
...
-	} else {
+	}
+	else {
 		scrape(doc, url);
 	}
 }
 
 
-function scrape(doc, url) {
+function scrape(doc) {
```
Comment (zuphilip) [inline Thieme.js]: _url

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**